controllers/host/host_controller.go

```diff
@@ -2277,6 +2277,38 @@ func (r *HostReconciler) UpdateStatusForFactoryInstall(
 	return nil
 }
 
+func (r *HostReconciler) isDayConfigComplete(instance *starlingxv1.Host) bool {
+	// TODO describe the reason
+	isFactory, err := r.GetFactoryInstall("deployment")
```

Comment:
Can we avoid hard-coded namespace here? I know currently there's an issue in the playbook with the configMap, which is only set in the deployment namespace. We can add a TODO here if it need to be hard-coded.

In longer term, we should apply the configMap for factory install in the system resource's namespace(as it is requested)